pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -618,10 +618,10 @@ def _check_adjdiff_supported_measurements(measurements: List[MeasurementProcess]
         @staticmethod
         def _check_adjdiff_supported_operations(operations):
             """Check Lightning adjoint differentiation method support for a tape.
-            Raise ``QuantumFunctionError`` if ``tape`` contains not supported measurements,
-            observables, or operations by the Lightning adjoint differentiation method.
             Args:
                 tape (.QuantumTape): quantum tape to differentiate.
+            Raise:
+                ``QuantumFunctionError`` if ``tape`` contains not supported measurements, observables, or operations by the Lightning adjoint differentiation method.
```

Comment:
With google style docstrings (I think we're using that right?), the raise section should read

```suggestion
            Raises:
                QuantumFunctionError: If ``tape`` contains not supported measurements, observables, or operations by the Lightning adjoint differentiation method.
```

Reply:
Correct! Let me fix it. Thanks